File: src/test/java/com/kyle/StudentTest.java
package com.kyle;

import com.kyle.classroom.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void averageTest(){
        Student student= new Student("curry",80,70);
        Assertions.assertEquals((80+70)/2,student.getAverage());
    }

    @Test
    public void highestTest(){
        Student student= new Student("curry",80,70);
        Assertions.assertEquals(80,student.hignest());
    }
}

File: src/test/kotlin/com/kyle/StudentTest.kt
package com.kyle.kotlin

import com.kyle.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestScoreTest(){
        val student = Student("curry",80,60)
        Assertions.assertEquals(80,student.highest())
    }
    @Test
    fun gradintTest(){
        val student = Student("curry",80,60)
        Assertions.assertEquals('C',student.gradint())
    }
    @Test
    fun passOrFail(){
        val student = Student("curry",80,60)
        Assertions.assertEquals("PASS",student.passOrFailed())
    }

}